src/main/java/uk/gov/hmcts/reform/idam/service/UserRoleService.java

```diff
@@ -34,7 +36,10 @@ public List<String> filterUsersWithRoles(List<String> staleUsers) {
         final RoleAssignmentResponse response;
         try {
             response = roleAssignmentClient.getRoleAssignments(
```

Comment:
We need to add s2s token as well - https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1496583644&__ncforminfo=tHbhf9Ra5JqdJHdz2ws7_Jkq5CaylcLhuSi7EDJ489py_xPcGo-RWiGMopxDxsND0NqT-tI6NdoEJpWeSrv88IJqvnPUpCZ8

Reply:
This part is still being done by interceptor.

Reply:
I prefer to have all the headers in one place, ideally, we should either have all the headers passed through the interceptor or via feign client.